clients/unshaded/src/main/java/tachyon/client/file/options/SetStateOptions.java

```diff
@@ -45,6 +46,17 @@ public Builder setPinned(boolean pinned) {
     }
 
     /**
+     * @param ttl the ttl(time to live) value (in milliseconds) to use; it specifies how long should
```

Comment:
let's be consistent with the description in `CreateOptions#setTTL`: "the TTL (time to live) value to use; it identifies duration (in milliseconds) the created file should be kept around before it is automatically deleted, no matter whether the file is pinned"